settings.gradle

```diff
@@ -1,3 +1,4 @@
-include ':logstash-core', 'logstash-core-benchmarks'
+include ':logstash-core', 'logstash-core-benchmarks', 'ingest-json-to-grok'
```

Comment:
Just make the top level as `ingest-converter` since we'll be dealing with more than just grok here.